Makefile

```diff
@@ -54,3 +55,10 @@ black-apply: # Apply changes with 'black'
 
 ruff-apply: # Resolve 'fixable errors' with 'ruff'
 	pipenv run ruff check --fix .
+
+# Development commands
+start-minio-server:
+	docker compose --env-file .env -f $(MINIO_COMPOSE_FILE) up -d
+
+stop-minio-server:
+	docker compose -f $(MINIO_COMPOSE_FILE) stop
```

Comment:
I think perhaps this needs the `.env` file as well?

```
docker compose --env-file .env -f $(MINIO_COMPOSE_FILE) stop
```

Reply:
Yep! Needed it. 😅 Updated!